api/control/v1alpha1/types.go

```diff
@@ -268,11 +268,15 @@ type BindingList struct {
 // +genclient:nonNamespaced
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Cluster,shortName={ct},categories={all}
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="WGROUP",type="string",JSONPath=".spec.apiGroup"
+// +kubebuilder:printcolumn:name="WRSC",type="string",JSONPath=".spec.resource"
 type CustomTransform struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec CustomTransformSpec `json:"spec,omitempty"`
+	Spec   CustomTransformSpec   `json:"spec,omitempty"`
```

Comment:
Sorry I was out the week this was added. 3 points:

1. What do you think about making this API's name consistent with other APIs we have, and with similar APIs in other projects, by adding a Policy postfix?
2. Why are we deviating from the resource selectors present in other places? I think it would be ideal to have a uniform API for resource selection, across API resources that require the capability.
3. I understand that the focus on providing only a `Remove` option is because there is already a user requirement for it, but it's strange that we don't refer to adding or updating anywhere. Do you think it can be added to the API and implemented later? Or at least have the intention documented.

Reply:
1. For "BindingPolicy" vs "Binding", I see a functional need for putting "Policy" in the name of one of the types. For CustomTransform, I do not see a functional need. I also think that just those two words convey the meaning pretty well. In mathematics, for example, we have things like "Fourier Transform". You may think of it as a policy for changing mathematical objects, but it is not necessary to say "Fourier Transform Policy".
2. We considered that earlier and decided to start simple and add complexity later if we found the need.
3. Yes, we explicitly decided to start simple with something that can be incrementally extended later if the need arises.

Reply:
(1) @vMaroon I think Policy is not a good fit here. @MikeSpreitzer Why not ResourceTransform to indicate what do we transforming ?   Do we have a non "Custom" transformation somewhere that this needs to be distinctly different ?  
(3) I mentioned before, I think we do need to add comment on that in our docs when we document this new feature. We all agree that for the release we are going to have this feature documented.

Reply:
@ezrasilvera: thank you for your LGTM on #2045, which defined the API here. I hope we do not have to rehash the "Kind" defined there now.

Yes, I think we are all agreed that documentation is a must-have.